src/products/android-l10n/removing_projects.md

```diff
@@ -26,13 +26,8 @@ This will ensure that the project stops syncing between Pontoon and the GitHub a
 * Remove corresponding project lines contained within the [config.toml](https://github.com/mozilla-l10n/android-l10n/blob/master/config.toml)
 * Remove corresponding project lines contained within the [l10n.toml](https://github.com/mozilla-l10n/android-l10n/blob/master/l10n.toml)
 
-Example PR that does all of this for Fire TV: https://github.com/mozilla-l10n/android-l10n/pull/289
-
-## Remove the project tag in Pontoon
-
-Go to the [admin view of android-l10n](https://pontoon.mozilla.org/admin/projects/android-l10n/), scroll to the bottom of the page, and remove the product under the “Tags” section.
-This ensures that the project will not be listed under “Tags” along with other active android-l10n projects in Pontoon.
+See [example PR](https://github.com/mozilla-l10n/android-l10n/pull/289) that does all of this for Fire TV.
 
 ## Re-enable sync in Pontoon
```

Comment:
This section needs to be removed.

Since it's only one project that needs to be "removed", and the files are gone at this point, we don't need to re-enable sync. This was necessary when the project was part of the old mono-project.

Reply:
Removed.